Octokit/Models/Response/RepositoryStar.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Represents additional information about a star (such as creation time)
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class RepositoryStar
+    {
+        public RepositoryStar() { }
```

Comment:
Ah, gotcha. I just assumed they were needed by the model deserializer. If it can instantiate models without the empty public ctor, I'm fine removing them.